.github/workflows/jekyll-gh-pages.yml

```diff
@@ -0,0 +1,51 @@
+# Sample workflow for building and deploying a Jekyll site to GitHub Pages
+name: Deploy Jekyll with GitHub Pages dependencies preinstalled
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["master"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
I don't see a value for this key. Is this valid YAML?

Hmm, maybe it [is valid](https://stackoverflow.com/a/836370/10840). Next question, is `workflow_dispatch: null` valid input for github actions? I'm guessing this was copied from somewhere, so probably?

Reply:
Yeah, I think this is valid. This was auto-created when I clicked on the button linked in the github [docs](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site).